Gems/Terrain/Assets/Shaders/Terrain/TerrainPBR_ForwardPass.azsl

```diff
@@ -155,7 +142,7 @@ ForwardPassOutput TerrainPBR_MainPassPS(VSOutput IN)
     {
         GatherSurfaceDataFromMaterials(surface.position.xy, detailFactor, macroColor, macroNormal, hasDetailSurface, detailSurface);
     }
-
+    
```

Comment:
Nitpick: Extra whitespace here

Reply:
nice catch, I'll get that in my next PR though